### PR TITLE
axolotl: remove unlock steps, verify recovery, implement fastbootd user action

### PR DESCRIPTION
**Pull request by amartinz**

### CODE REVIEW

File: v2/devices/axolotl.yml [hunk 1]
```diff
@@ -4,38 +4,29 @@ formfactor: "phone"
...
-    description: 'Your device must be "unlocked" in order to be able to flash it. Unlocking the bootloader will delete your personal data!<br><br>If your bootloader is already unlocked, you are good to go. Otherwise please check the link below (More...) and follow the instructions.'
-    link: "https://wiki.lineageos.org/devices/axolotl/install#unlocking-the-bootloader"
   bootloader:
     title: "Reboot to Bootloader"
     description: "With the device powered off, hold Volume Up + Power."
     image: "phone_power_up"
     button: true
+  fastbootd:
+    title: "Reboot to Fastbootd"
+    description: "With the device powered off, hold Volume Down + Power to boot into recovery.\n\nInside the recovery, press on 'Advanced' and select 'Enter fastboot'."
```
Comment (amartinz) [inline v2/devices/axolotl.yml:14]: This step is generic for Halium 10 devices.
Should we implement that common somewhere?

I think we should try to reduce user visible text in device configs as much as possible to make it easier for translation.

**Comment by amartinz** [comment]
I just got a failure trying to assert recovery with `0.9.5-beta`:

```
verbose: running adb action assert_prop
error: Error: adb:assert_prop: Error: unknown getprop error
stack trace: Error: unknown getprop error
    at /tmp/.mount_ubportL2xeIJ/resources/app.asar.unpacked/node_modules/promise-android-tools/lib/module.cjs:565:21
    at /tmp/.mount_ubportL2xeIJ/resources/app.asar.unpacked/node_modules/promise-android-tools/node_modules/cancelable-promise/dist/CancelablePromise.js:27:18
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

**Comment by amartinz** [comment]
updated prebuilt recovery, which fixes above assertion error and installs successfully.

open-cuts report: https://ubports.open-cuts.org/run/623de8a4bfb7110008fb5360

**Review by Flohack74 — APPROVED** [review]

**Comment by NeoTheThird** [comment]
> I forgot to update the prebuilt recovery used by the installer, which includes ro.ubports.recovery instead of ro.ubuntu.recovery.

But which will be used in the wild?

**Comment by amartinz** [comment]
> > I forgot to update the prebuilt recovery used by the installer, which includes ro.ubports.recovery instead of ro.ubuntu.recovery.
> 
> But which will be used in the wild?

the one specified in installer only.

**Pull request merged**